src/components/LogEntryList/LogEntryList.js

```diff
@@ -2,15 +2,19 @@ import LogEntry from '../LogEntry/LogEntry';
 import styled from 'styled-components';
 import { Header } from '../styled-components/Header';
 // check if there are logEntries?
-export default function LogEntryList({ logEntries }) {
+export default function LogEntryList({ logEntries, onDelete }) {
   return (
     <Wrapper>
       <Header id="header">your log entries</Header>
       <EntryList role="list" aria-labelledby="header">
-        {logEntries.map((logEntryData, index) => {
+        {logEntries.map(logEntryData => {
           return (
-            <li key={index}>
-              <LogEntry logEntryData={logEntryData} />
+            <li key={logEntryData._id}>
+              <LogEntry
+                logEntryData={logEntryData}
+                onDelete={onDelete}
+                _id={logEntryData._id}
```

Comment:
You could save yourself the trouble of passing the `_id` as prop and just pass it to the `onDelete` function here.
```suggestion
                onDelete={() => onDelete(logEntryData._id)}
```
In your `LogEntry` component you'd then just write:
```js
      <Button
        type="button"
        variant="invisible"
        onClick={onDelete}
        aria-label="Delete this log entry"
      >
        <GrTrash />
      </Button>
```